content/zh-cn/blog/_posts/2022-05-20-non-graceful-node-shutdown.md

```diff
@@ -0,0 +1,220 @@
+除非 Pod 上设置有与之匹配的容忍度。原来挂接到已关闭节点的持久卷（Persistent volume）将被解除挂接，
+对于 StatefulSet，系统将在不同的运行节点上成功创建替换 Pod。
+
+```
+$ kubectl taint nodes <node-name> node.kubernetes.io/out-of-service=nodeshutdown:NoExecute
+
+$ kubectl get pod -o wide
+NAME    READY   STATUS    RESTARTS   AGE    IP           NODE                      NOMINATED NODE   READINESS GATES
+web-0   1/1     Running   0          150m   10.244.2.4   k8s-node-876-1639279816   <none>           <none>
+web-1   1/1     Running   0          10m    10.244.1.7   k8s-node-433-1639279804   <none>           <none>
+```
+
+<!--
+Note: Before applying the out-of-service taint, you **must** verify that a node is already in shutdown or power off state (not in the middle of restarting), 
+either because the user intentionally shut it down or the node is down due to hardware failures, OS issues, etc.
+-->
+注意：在应用 out-of-service 污点之前，你**必须**确认节点是否已经处于关闭或断电状态（不是在重新启动过程中），
+节点关闭的原因可能是用户有意将其关闭，也可能是节点由于硬件故障、操作系统问题等而关闭。
+
+<!--
+Once all the workload pods that are linked to the out-of-service node are moved to a new running node, and the shutdown node has been recovered, you should remove
+that taint on the affected node after the node is recovered.
+If you know that the node will not return to service, you could instead delete the node from the cluster.
+-->
+一旦关联到无法提供服务的节点的所有工作负载 Pod 都被移动到新的运行中的节点，并且关闭了的节点也已恢复，
+你应该在节点恢复后删除受影响节点上的污点。如果你知道该节点不会恢复服务，则可以改为从集群中删除该节点。
+
+<!--
+## What’s next?
+
+Depending on feedback and adoption, the Kubernetes team plans to push the Non-Graceful Node Shutdown implementation to Beta in either 1.25 or 1.26.
+-->
+## 下一步是什么？
+
+根据反馈和采用情况，Kubernetes 团队计划在 1.25 或 1.26 版本中将节点非体面关闭实现推送到 Beta 阶段。
+
+<!--
+This feature requires a user to manually add a taint to the node to trigger workloads failover and remove the taint after the node is recovered. 
+In the future, we plan to find ways to automatically detect and fence nodes that are shutdown/failed and automatically failover workloads to another node.
+-->
+此功能需要用户手动向节点添加污点以触发工作负载故障转移，并要求用户在节点恢复后移除污点。
+未来，我们计划寻找方法来自动检测和隔离已关闭的或已失败的节点，并自动将工作负载故障转移到另一个节点。
+
+<!--
+## How can I learn more?
+
+Check out the [documentation](/docs/concepts/architecture/nodes/#non-graceful-node-shutdown)
+for non-graceful node shutdown.
+-->
+## 怎样才能了解更多？
+
+查看节点非体面关闭相关[文档](/zh-cn/docs/concepts/architecture/nodes/#non-graceful-node-shutdown)。
+
+<!--
+## How to get involved?
+-->
+## 如何参与？
+
+<!--
+This feature has a long story. Yassine Tijani ([yastij](https://github.com/yastij)) started the KEP more than two years ago. 
+Xing Yang ([xing-yang](https://github.com/xing-yang)) continued to drive the effort. 
+There were many discussions among SIG Storage, SIG Node, and API reviewers to nail down the design details. 
+Ashutosh Kumar ([sonasingh46](https://github.com/sonasingh46)) did most of the implementation and brought it to Alpha in Kubernetes 1.24.
+-->
+此功能特性由来已久。Yassine Tijani ([yastij](https://github.com/yastij)) 在两年多前启动了这个 KEP。
```

Comment:
replace the brackets with Chinese one, same below at line 182 to 184 and line 193 to 200
```suggestion
此功能特性由来已久。Yassine Tijani（[yastij](https://github.com/yastij))）在两年多前启动了这个 KEP。
```